src/api/c/graphics_common.cpp

```diff
@@ -366,6 +424,24 @@ forge::VectorField* ForgeManager::getVectorField(forge::Chart* chart, int nPoint
     return mVcfMap[keypair];
 }
 
+bool ForgeManager::getChartAxesOverride(forge::Chart* chart)
+{
+    ChartAxesOverrideIter iter = mChartAxesOverrideMap.find(chart);
+    if (iter == mChartAxesOverrideMap.end()) {
+        AF_ERROR("Chart Not Found!", AF_ERR_ARG);
```

Comment:
Isn't this error code reserved for API level arguments ? Charts are not exposed to user and the current callee is also not exposed externally. Therefore, i think it should be `AF_ERR_INTERNAL`. The string message can leave more clues if needed.